settings.gradle

```diff
@@ -34,10 +34,12 @@ dependencyResolutionManagement {
             library("sonar-ws", "org.sonarsource.sonarqube", "sonar-ws").version(sonarqubeVersion)
             library("slang-antlr", "org.sonarsource.slang", "slang-antlr").version(slangDependenciesVersion)
             library("slang-testing", "org.sonarsource.slang", "slang-testing").version(slangDependenciesVersion)
-            library("mockito-core", "org.mockito", "mockito-core").version("5.13.0")
+            library("mockito-core", "org.mockito", "mockito-core").version("5.14.2")
             library("assertj-core", "org.assertj", "assertj-core").version("3.26.3")
-            library("junit-jupiter-api", "org.junit.jupiter", "junit-jupiter-api").version("5.11.0")
-            library("junit-jupiter-engine", "org.junit.jupiter", "junit-jupiter-engine").version("5.11.0")
+            library("junit-jupiter-api", "org.junit.jupiter", "junit-jupiter-api").version("5.11.3")
+            library("junit-jupiter-engine", "org.junit.jupiter", "junit-jupiter-engine").version("5.11.3")
+            library("junit", "junit", "junit").version("4.13.2")
+            //testImplementation group: 'junit', name: 'junit', version: '4.13.2'
```

Comment:
Can we make these library upgrades in a separate PR? Just to make sure we don't run into any issue before release?

Reply:
it is done, I forgot to remove them with the others